src/components/Form/FormFields/WeekCronSelect.vue

```diff
@@ -181,11 +181,8 @@ export default {
     // 计算索引
     countIndex(val) {
       const one = val.substr(0, 2)
-      const a1 = one.startsWith('0') ? one.substr(1, 2) : one
-      var reg = RegExp(/30/)
-      const a2 = val.match(reg) ? 1 : 0
-      const curIndex = (a1 * 2) + a2
-      return curIndex
+      const index = one.startsWith('0') ? one.substr(1, 2) : one
+      return Number(index)
     },
     formatDate(date, fmt) {
       const o = {
```

Comment:
Your code seems to be written in JavaScript. It includes two functions `countIndex` and `formatDate`. However, there haven't been any specific coding errors reported here.

The function names suggest they might refer to something different from your other examples that seem to involve programming of HTML or CSS. If you specify more details about what these parts represent or need assistance with, I'd be more than happy to help further!